contributed_definitions/nyaml/NXcalibration.yaml

```diff
@@ -13,6 +13,25 @@ NXcalibration(NXobject):
   description(NX_CHAR):
     doc: |
       A description of the procedures employed.
+  start_time(NX_DATE_TIME):
+    doc: |
+      The start time of the calibration.
+  end_time(NX_DATE_TIME):
+    doc: |
+      The end time of the calibration.
+  calibration_interval(NX_FLOAT):
+    unit: NX_TIME
+    doc: |
+      The time difference between the start and end time of the calibration.
```

Comment:
This is the docs and just a clarification. It is clear that either some use their own clock or may come from an instrument.

Reply:
Why have calibration_interval at all? Any software can get it from the start and end time.

Reply:
My point is that this is redundant information if you have the start and end time of the calibration and writing it could lead to errors if the interval is not the same. But you can keep it as well.

Reply:
A agree with you about the possible conflict. it is there if only time data is about `interval` of the calibration, without the `start` and `end` values.